tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -339,3 +340,24 @@ Check External Links Status
         ${status}=  Check HTTP Status Code   link_to_check=${href}
         Log    ${idx}. ${href} gets status code ${status}
     END
+
+Verify Cluster Settings Is Available
+    [Documentation]    Verifies submenu Settings > Cluster settings" is visible
+    Page Should Contain    Settings
+    Menu.Navigate To Page    Settings    Cluster settings
+    Capture Page Screenshot
+    Wait Until Page Contains    Update global settings for all users    timeout=30
+    Wait Until Page Contains Element    ${USAGE_DATA_COLLECTION_XP}    timeout=30
+
+Verify Cluster Settings Is Not Available
+    [Documentation]    Verifies submenu Settings > Cluster settings is not visible
+    ${cluster_settings_available}=    Run Keyword And Return Status    Verify Cluster Settings Is Available
+    Should Be True    not ${cluster_settings_available}    msg=Cluster Settings shoudn't be visible for this user
```

Comment:
would it be possible to use `Should Not Be True` to be consistent with what you did in ODS keywords